.github/workflows/pr-build-and-e2e-tests.yml

```diff
@@ -1,7 +1,9 @@
 name: Run tests against PR
 on:
-    pull_request:
     workflow_dispatch:
+    pull_request:
+        paths-ignore:
+            - '**/changelog/**'
```

Comment:
What if this is a "required" check and is skipped? Can the PR still be merged?

Reply:
@psealock yeah of course. Please see my example PRs.

Reply:
>When they are skipped its ok.

I don't think thats right. I created a changelog only PR on my fork after having added a "required" status to one of the checks. Because the check is required, its saying `Expected — Waiting for status to be reported` but that status will never be reported because the check will never be run.

<img width="1008" alt="image" src="https://user-images.githubusercontent.com/1922453/210671152-0c365fe4-780f-46ee-9eaa-7b701ed796e7.png">

Reply:
Damn man I misread or misunderstood the docs on this :/ that's really dumb.

It looks like we will have to move this to a conditional instead https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks

My bad @psealock and thanks for spotting that